app/actions/contact_merge_action.rb

```diff
@@ -0,0 +1,30 @@
+class ContactMergeAction
+  attr_reader :base_contact, :mergee_contact
+
+  def initialize(params)
```

Comment:
We are using this gem `attr_extras`. You can remove this block and use `pattr_initialize [:base_contact_id!, :mergee_contact_id]`